src/contributing.md

```diff
@@ -47,5 +47,5 @@ thus we may take considerable time to get back to you.
     
     Our works are attributed to "The Rand Project Developers". This is not a
     formal entity but merely the collection of all contributors to this project.
-    For more, see the [COPYRIGHT](COPYRIGHT) file.
+    For more, see the `COPYRIGHT` file.
```

Comment:
Why did you remove the link?

Reply:
Because it was broken. You can try it in the current version of the book. We don't publish this file, so we could only link back to the repo.